hack/templates/aws_v1alpha1_configmap.tmpl

```diff
@@ -3,6 +3,7 @@ kind: Template
 parameters:
 - name: ROOT 
 - name: BASE
+- name: ACCOUNTLIMIT
```

Comment:
On a similar note, won't this will break the existing `create-ou-map` target, which doesn't set `ACCOUNTLIMIT`?

If we leave this out, is the effect that the configmap gets the "nil" value of zero for the account limit? And does that behave improperly for test environments?

Reply:
I'd say change `create-ou-map` to include this and make the target required for `predeploy`.